examples/flows/hello-world-yaml/definition.yaml

```diff
@@ -0,0 +1,14 @@
+Comment: Run the Hello World Action as the Flow's identity
+StartAt: RunHelloWorld
+States:
+  RunHelloWorld:
+    RunAs: Flow
+    Comment: Say Hello World
+    Type: Action
+    ActionUrl: 'https://actions.automate.globus.org/hello_world'
+    ActionScope: 'https://auth.globus.org/scopes/actions.globus.org/hello_world'
+    Parameters:
+      echo_string.$: $.input.echo_string
```

Comment:
Not a comment on this PR, just on the proposed approach of using YAML, the mixing of JSON path into the YAML feels slightly weird to me, but I'm not proposing anything different unless a) someone knows of a YAML equivalent to JSON path and 2) it can easily be translated to JSON Path. Given that this is referencing the Flow state, and the Flow state's "native" representation is JSON, I doubt this is even a good idea (introducing some hypothetical YAML-style path syntax), but I thought I'd mention to see if it elicits any thoughts.

Reply:
Just leaving a comment to say the mixing is sorta lame but I don't know of any YAML-path. If I squint real hard I can sorta mentally map the path onto a json input but I can't say it'll work in every scenario.

Reply:
So just to clarify, since YAML is a superset of JSON, the JSON path IS the YAML path. So it's not technically mixing anything but by supporting YAML we are introducing some confusion and I can see how that could cause problems. Not sure what the best thing to do is besides just document it really well. I'm open to any solution